doc/server/manual-sphinx/historyGET.rst

```diff
@@ -0,0 +1,117 @@
+=========================
+GET-History
+=========================
+ **Please refer when you Get a History.**
+Request
+=======
+
+Path
+----
+.. list-table::
+   :header-rows: 1
+
+   * - URL
+     - Comments
+   * - /history
+     - N/A
+
+Parameters
+----------
+.. list-table::
+   :header-rows: 1
+
+   * - Parameter
+     - Value
+     - Comments
+     - JSON
+     - JSONP
+   * - fmt
+     - json or jsonp
+     - This parameter is omitted, the return format is json.
```

Comment:
It seems that this sentences should have `If` clause like this:

If this parameter is omitted, the return format is json.